packages/block-editor/src/components/block-title/use-block-display-title.js

```diff
@@ -71,7 +72,12 @@ export default function useBlockDisplayTitle( clientId, maximumLength ) {
 		? getBlockLabel( blockType, attributes )
 		: null;
 
-	const label = reusableBlockTitle || blockLabel;
+	const sectionTitle =
+		hasBlockSupport( name, '__experimentalSection', false ) &&
+		attributes.metadata?.isSection &&
```

Comment:
Do we really need to check isSection here :) Why not just allow any blocks to have a name?